src/commands/create/index.ts

```diff
@@ -117,27 +126,22 @@ export default class Create extends Command {
     }
 
     private async _runInteractiveMode(flags: any) {
-        let interactiveResponses: any = await inquirer.prompt([
-            Create.prompts.name,
-            Create.prompts.language,
-            Create.prompts.exampleQuestion,
-        ]);
+        const interactiveResponses: any = await inquirer.prompt([Create.prompts.language, Create.prompts.exampleQuestion]);
 
-        flags.name = interactiveResponses.name;
         flags.language = interactiveResponses.language;
         flags.example = interactiveResponses.exampleQuestion;
 
         if (flags.example) {
             availableExamples = this._filterAvailableExamplesByLanguage(flags.language);
-            interactiveResponses = await inquirer.prompt([Create.prompts.exampleUse]);
+            const exampleResponse = await inquirer.prompt([Create.prompts.exampleUse]);
+            flags.example = flags.name = exampleResponse.exampleUse;
         } else {
-            interactiveResponses = await inquirer.prompt([Create.prompts.interface]);
-        }
-        flags.example = interactiveResponses.exampleUse;
-        flags.interface = interactiveResponses.interface;
-
-        if (flags.interface && flags.interface.length > 0) {
-            await this._handleAdditionalInterfaceArgs(flags.interface);
+            const interactiveNoExampleResponses: any = await inquirer.prompt([Create.prompts.name, Create.prompts.interface]);
```

Comment:
Better than calling this "noExamples" (so an inversion of the previous block) its easier to read and understand when you call it what it actually is: Responses for a skeleton or empty app
```suggestion
            const interactiveSkeletonAppResponses: any = await inquirer.prompt([Create.prompts.name, Create.prompts.interface]);
```